go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/onflow/flow-evm-gateway
 
-go 1.20
+go 1.22
```

Comment:
is this necessary? it's a good practice to do this sort of upgrade separately from a feature change in case you have to rollback.

Reply:
wasn't aware this was done, will revert